hack/ovn-kind-cno.sh

```diff
@@ -198,18 +183,14 @@ spec:
   - ${SERVICE_NETWORK}
 EOF
 
-count=1
-until kubectl get pod -n openshift-ovn-kubernetes -o jsonpath="{.items[0].status.phase}" 2> /dev/null | grep Running &> /dev/null;do
-  if [ $count -gt 15 ]; then
-    echo "OVN-k8s pods are not running"
-    exit 1
-  fi
-  echo "OVN pod not available yet on attempt $count"
-  count=$((count+1))
-  sleep 10
-done
-sleep 10
+# Configuring secret for multus-admission-webhook
+# https://raw.githubusercontent.com/openshift/multus-admission-controller/master/hack/webhook-create-signed-cert.sh
+$CNO_PATH/hack/webhook-create-signed-cert.sh --service multus-admission-controller --namespace openshift-multus --secret multus-admission-controller-secret
```

Comment:
multus admission webhook was failing because it was trying to mount the certificates from this secret, don't know where this has to be fixed though